### ISSUE TITLE
[Feature Request] Add an additional keep tag

### ISSUE THREAD

**Issue by Korrasera** [DestinyItemManager/DIM#8717]
### Proposed change

Add a 6th tag to the tagging system that is similar but visually different from the current Keep tag, so that there would be three 'primary' tags, Keep 1, Keep 2, and Favorite. The names are irrelevant, you could have one be a DIM orange Keep tag and one be a DIM blue Keep tag and that would work just as well.

### How does this fit into your workflow?

This would improve the functionality of the tagging system allow us to easily differentiate gear with a simple glance at DIM. 

Currently, we can only tag our preferred items as Keep or Favorite. This system is useful, but limited. It only gives us two categories to work with, which means it doesn't really handle duplicate items very well.

Since Destiny is built around both PvE and PvP scenarios, it's common for players to have duplicate items meant to handle different situations. Using Keep and Favorite to handle this visually is possible, but doing so limits the functionality of both tags. As an example, instead of relying on Favorite to tag the items I actually prefer, I use it to indicate what items I'm keeping for PvP, while I tag items Keep if I'm going to use them in PvE content.

My intended use case scenario for this change request is to use one Keep tag for PvE gear and one Keep tag for PvP gear, while reserving the Favorite tag for things I actually want to favorite, like god-rolls and special cases like crafted weapons being leveled or gear that's particularly useful in the current season.

As I understand it, the point behind the tagging system was two-fold; to create a system that would let you easily tag items you were planning to keep or discard or infuse, but also to allow you to visually differentiate all of the items sitting in your vault. The Archive tag is a perfect example, as it lets us designate an item that we want to keep but isn't important anymore and should be left out of most searches. Adding this additional tag would improve this functionality without making the app more complex or difficult to use.

To address some concerns that I've already thought of:

- Clearing the tag from an item isn't a substitute for this functionality. All gear comes into the app untagged, so you can't strip an item of a tag as an alternative to tagging it with keep or favorite without interfering with the core loot cycle that DIM is trying to support.
- Notes aren't a substitute for this functionality as they aren't visible on the icon for the item, so they can't visually differentiate the pieces at a glance.
- Using Favorite as a 2nd form of the Keep tag means not being able to use it for things you'd actually like to favorite.

Long story short:

Destiny is a game that's bifurcated between PvE and PvP spaces. I think we need a 2nd keep tag that we can use to demonstrate this difference in DIM as well.

**Comment by Korrasera** [comment]
As an alternative suggestion, this could also be accomplished by letting us 'color' a tag on an item, because that would maintain the meaning of each of the current tags while also allowing us to visually indicate different kinds of gear. It would effectively add a 2nd axis for the tagging system.

It could be as simple as adding another color (DIM orange or DIM blue are an obvious choice, provided the contrast works well for icons) and a tag filter for color, like tagcolor:green or the like.

**Comment by bhollis** [comment]
We plan on directly supporting keep-pvp and keep-pve tags.
